Lib9c/Action/ItemEnhancement.cs

```diff
@@ -268,6 +268,9 @@ public override IAccountStateDelta Execute(IActionContext context)
             enhancementEquipment.Unequip();
 
             enhancementEquipment = UpgradeEquipment(enhancementEquipment);
+
+            var requiredBlockIndex = ctx.BlockIndex + 1;
```

Comment:
How about defining a new constant such as `requiredBlocks` and set the value as 1 for further changes?